kadi/commands/commands.py

```diff
@@ -184,14 +192,19 @@ def _find(start=None, stop=None, **kwargs):
     if date:
         start = DateTime(date).date  # clip resolution to nearest msec
         stop = DateTime(start) + 0.001 / 86400  # exactly 1 msec later
+        inclusive_stop = False
```

Comment:
This makes sense to work with the logic below for inclusive_stop, but would there be value in just using some kind of "==" somewhere for the special case instead of generalizing?

Reply:
Yup, done.